<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mano_Animatronica_Aplicacion</class>
 <widget class="QMainWindow" name="Mano_Animatronica_Aplicacion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mano_Animatronica_Aplicacion</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushB_Cargar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Cargar 
 Movimientos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushB_PlayPause">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>80</y>
      <width>171</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Play / Pause</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="grB_Modo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>221</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Modo de Operaci√≥n</string>
    </property>
    <widget class="QComboBox" name="cBx_Modo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Seleccione Modo</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Guante -&gt; PC</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PC -&gt; Mano</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Guante -&gt; Mano</string>
      </property>
     </item>
    </widget>
    <widget class="QRadioButton" name="rdB_Guante_PC">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Guante -&gt; PC</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rdB_PC_Mano">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>PC -&gt; Mano</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rdB_Guante_Mano">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Guante -&gt; Mano</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grB_Sliders">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>220</y>
      <width>281</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Sliders</string>
    </property>
    <widget class="QSlider" name="sld_Mayor">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>59</y>
       <width>20</width>
       <height>171</height>
      </rect>
     </property>
     <property name="maximum">
      <number>140</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="invertedAppearance">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QSlider" name="sld_Anular">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>59</y>
       <width>20</width>
       <height>171</height>
      </rect>
     </property>
     <property name="maximum">
      <number>140</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="invertedAppearance">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QSlider" name="sld_Menor">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>59</y>
       <width>20</width>
       <height>171</height>
      </rect>
     </property>
     <property name="maximum">
      <number>120</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="invertedAppearance">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QSlider" name="sld_Pulgar">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>59</y>
       <width>16</width>
       <height>171</height>
      </rect>
     </property>
     <property name="maximum">
      <number>120</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="invertedAppearance">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QSlider" name="sld_Indice">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>59</y>
       <width>16</width>
       <height>171</height>
      </rect>
     </property>
     <property name="maximum">
      <number>140</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="invertedAppearance">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_Sliders">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>281</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string> Menor     Anular     Mayor      Incide      Pulgar</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grB_Valores">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>671</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Valores actuales</string>
    </property>
    <widget class="QLabel" name="lbl_VMenor">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Menor: 9999</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_VAnular">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Anular: 9999</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_VMayor">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Mayor: 9999</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_VIndice">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>20</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Indice: 9999</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_VPulgar">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>20</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Pulgar: 9999</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grB_Mano">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>50</y>
      <width>241</width>
      <height>411</height>
     </rect>
    </property>
    <property name="title">
     <string>Mano</string>
    </property>
    <widget class="QLabel" name="lbl_1_Base_Img">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>216</width>
       <height>384</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_2_Menor_Img">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>216</width>
       <height>384</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_3_Anular_Img">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>216</width>
       <height>384</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_4_Mayor_Img">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>216</width>
       <height>384</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_5_Indice_Img">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>216</width>
       <height>384</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_6_Pulgar_Img">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>216</width>
       <height>384</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="lbl_Archivo">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre_archivo.hnd</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushB_Guardar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Guardar 
 Movimientos</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
